js/tweek-client/src/TweekFetchError.ts

```diff
@@ -0,0 +1,12 @@
+export class TweekFetchError extends Error {
```

Comment:
Don't we already have a FetchError class? 🤔

Reply:
https://github.com/Soluto/tweek-clients/blob/master/js/tweek-client/src/FetchError.ts

Reply:
yes but for `FetchError` class we need `response` object, which we don't have

Reply:
You actually do, because you create it when checking for response.ok otherwise you just have an error object.
Or am I missing something?

Reply:
In case fetch throws I only have Error
https://github.com/Soluto/tweek-clients/pull/159/files#diff-ce0dcec6235c35b2939a0cfab8f325a9R68

Reply:
ahh ok
I wanted to create a "nicer" error and send it to `onError`
Instead of making each client that uses tweek with `onError` to do something like the function `parseTweekErrorResponse` (especially this part: `await res.clone().text()`)

if u think it's too much I can remove `TweekFetchError` and use `FetchError`

Reply:
Or you can modify the existing error, I don't think there's a need for 2 different classes that essentially do the same

Reply:
I removed my error and use `FetchError`